libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTracker.java

```diff
@@ -690,6 +690,23 @@
         ENCRYPTED_LOGGING_UPLOAD_SUCCESSFUL,
         ENCRYPTED_LOGGING_UPLOAD_FAILED,
         READER_POST_REPORTED,
+        COMMENT_APPROVED,
+        COMMENT_UNAPPROVED,
+        COMMENT_SPAMMED,
+        COMMENT_UNSPAMMED,
+        COMMENT_FOLLOWED,
+        COMMENT_UNFOLLOWED,
+        COMMENT_LIKED,
+        COMMENT_UNLIKED,
+        COMMENT_TRASHED,
+        COMMENT_UNTRASHED,
+        COMMENT_REPLIED_TO,
+        COMMENT_EDITED,
+        COMMENT_VIEWED,
+        COMMENT_DELETED,
+        COMMENT_QUICK_ACTION_APPROVED,
+        COMMENT_QUICK_ACTION_LIKED,
+        COMMENT_QUICK_ACTION_REPLIED_TO,
```

Comment:
I think you did it on purpose, and not sure it's worth, but placing here as a reminder to discuss about it; we are missing a few events that are managed by iOS (unless they are not used anymore there, we should verify in tracks):

- followed_blog_notification_settings_comments_on
- followed_blog_notification_settings_comments_off

Also not sure it's worth here, but we should add to `stats_overview_type_tapped` a source as well (VIEWS/VISITORS/LIKES and COMMENTS). see pbArwn-1oJ-p2#comment-1969 for some more context.

Reply:
After doing some reasearch, It looks like we do track the `followed_blog_notifications_settings_comments_on` and `followed_blog_notifications_settings_comments_off` events in Android already.

We will tackle adding a source to the stats event in a separate PR 👍

Reply:
>After doing some reasearch, It looks like we do track the followed_blog_notifications_settings_comments_on and followed_blog_notifications_settings_comments_off events in Android already.

ahah 😄 , I was missing the typo with `s` in notifications! good catch :). Agree for the rest 👍